src/core/a-animation.js

```diff
@@ -356,13 +357,14 @@ function getAnimationValues (el, attribute, dataFrom, dataTo, currentValue) {
   var from = {};
   var partialSetAttribute;
   var to = {};
-
-  if (attributeSplit.length === 2) {
+  if (attributeSplit.length === 2 && attributeSplit.indexOf('color') === -1) {
```

Comment:
If the attribute is `color`, then the first clause will fail because there's nothing to split on. Thus you just move on to the `isNaN(dataTo)` and do color interpolation there.

If the attribute is `material.color`, then you know you can check the schema, right?